app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -10,38 +10,40 @@ import EmptyState from "components/Common/EmptyState";
 import { CONTACTS_DATA } from "./constants";
 import DeleteAlert from "./DeleteAlert";
 import MenuBar from "./MenuBar";
-import NewContactPane from "./Pane/NewContactPane";
+import NewContactPane from "./Pane/NewContact";
 import Table from "./Table";
 
 const Contacts = () => {
   const [searchTerm, setSearchTerm] = useState("");
   const [selectedContactIds, setSelectedContactIds] = useState([]);
-  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
-  const [showMenuBar, setShowMenuBar] = useState(false);
```

Comment:
The previous name that is `showMenuBar` and `showDeleteAlert` were perfectly valid names because they denote a boolean value. Keywords like `show`, `is`, `has`, `should` etc can denote boolean values. Ultimately most questions are boolean values.